Sources/FluentBenchmark/SolarSystem/Governor.swift

```diff
@@ -0,0 +1,70 @@
+import FluentKit
+
+public final class Governor: Model {
+    public static let schema = "governors"
+
+    @ID(key: .id)
+    public var id: UUID?
+
+    @Field(key: "name")
+    public var name: String
+
+    @Parent(key: "planet_id")
+    public var planet: Planet
+
+    public init() { }
+
+    public init(id: IDValue? = nil, name: String) {
+        self.id = id
+        self.name = name
+    }
+
+    public init(id: IDValue? = nil, name: String, planetId: UUID) {
+        self.id = id
+        self.name = name
+        self.$planet.id = planetId
+    }
+}
+
+public struct GovernorMigration: Migration {
+    public func prepare(on database: Database) -> EventLoopFuture<Void> {
+        database.schema("governors")
+            .field("id", .uuid, .identifier(auto: false))
```

Comment:
Add `.unique` and `.required` constraints here for completeness' sake.
```suggestion
            .field(.id, .uuid, .identifier(auto: false), .required, .unique)
```